servers/audio_server.h

```diff
@@ -473,6 +473,8 @@ class AudioServer : public Object {
 
 	PlaybackType get_default_playback_type() const;
 
+	Vector<Ref<AudioStreamPlayback>> sample_playback_list;
```

Comment:
This could be a `LocalVector` I think since it's not passed around.

Reply:
Updated!